group_vars/wingedhelix_cluster/vars.yml

```diff
@@ -447,12 +447,11 @@ lustre_client_networks:
     interface: enp5s0
 pfs_mounts:
   - pfs: umcgst12
-    device: /dev/vdb
-    source: "{{ ip_addresses[groups['nfs_server'][0]][stack_prefix + '_internal_storage']['address'] }}:/mnt"
+    source: "{{ ip_addresses['wh-sai'][stack_prefix + '_internal_storage']['address'] }}:/mnt"
```

Comment:
I see it was before source `/dev/vdb`, destination `/mnt`. But there are many cases when you mount more than 1 mount, and if that one is already taken by `/mnt` it can become a problem.

I would suggest to change the `:/mnt` into `/mnt/something`

Reply:
That is already the case. What is used in the playbook is `{{ source }}/{{ pfs }}/` dy design for the source and `/mnt/{{ pfs }}` for the destination. So in this case you would get `10.10.2.99:/mnt/umcgst12/` mounted in `/mnt/umcgst12/`; We never use `/mnt` as a point point or it is a bug.

Reply:
ah, then great. I misinterpreted the `:/mnt` as default location for no pfs provided. Where it is actually just a base for pfs to be attached to.